database/migrations/2020_06_20_213819_create_overclocks_stats_table.php

```diff
@@ -29,7 +29,7 @@ public function up()
             $table->string('text_description', 1000);
             $table->string('json_stats', 1000);
             $table->string('icon', 1000);
-            $table->bigInteger('patch_number_index');
+            $table->bigInteger('patch_id');
```

Comment:
Unfortunately, you will have to generate new migrations to fix this. Otherwise anyone that has migrated won't get the updates.